src/cleanlab_tlm/utils/rag.py

```diff
@@ -657,7 +657,10 @@ def __repr__(self) -> str:
     },
     {
         "name": "query_ease",
-        "criteria": "Determine whether the above User Request appears simple and straightforward. A bad User Request will appear either: disgruntled, complex, purposefully tricky, abnormal, or vague, perhaps missing vital information needed to answer properly. The simpler the User Request appears, the better. If you believe an AI Assistant could correctly answer this User Request, it is considered good. If the User Request is non-propositional language, it is also considered good.",
+        "criteria": """Determine whether the above User Request appears simple and straightforward.
+A bad User Request will appear either: ambiguous in intent, disgruntled, complex, purposefully tricky or abnormal.
+A good User Request is phrased clearly and expresses an achievable intent, even if it contains pronouns or references to prior context. Should an AI assistant be able to easily answer the User Request, it is considered good. Basic conversational and non-propositional statements are also considered good.
+Vagueness alone does not make a Request bad - since an AI assistant responsible for addressing the query could potentially access additional user context, be aware of domain-specific terms or access domain-specific knowledge bases to provide a proper response, so a User Request that seems to be missing information could still be good.""",
```

Comment:
[nit] less verbose, since AI assistant is mentioned above

```suggestion
Vagueness alone does not make a Request bad - since the AI assistant could potentially access additional user context, be aware of domain-specific terms or access domain-specific knowledge bases to provide a proper response, so a User Request that seems to be missing information could still be good.""",
```

Reply:
Re: 
> I made a slight wording change to the query_ease criteria in [7503bc0](https://github.com/cleanlab/cleanlab-tlm/pull/57/commits/7503bc08bcc4f67484653cc23ceae88cb3c1d6ae), just to make the phrasing more straightforward and reduce any confusion about vagueness automatically making a query “bad.” Since AI assistants often have access to user context or domain-specific knowledge, I wanted to reflect that nuance better in the description.

Can you link to benchmarks that show the improvement there?  I'm out-of-the-loop on what open-domain benchmarks have been run. Would prefer each benchmark run to be shared in slack with the findings on modifications.

Reply:
To me something like this seems slightly clearer:

Determine whether the above User Request appears simple and straightforward.
A bad User Request will appear either: ambiguous in intent, complex, purposefully tricky, abnormal, or disgruntled.
A good User Request is phrased clearly and expresses an achievable intent. Basic conversational and non-propositional statements are also considered good.
Should an AI Assistant be able to properly answer the User Request, it is considered good. The AI Assistant handling this User Request has additional knowledge about: the user, domain-specific terms and abbreviations, and any necessary factual information. So a User Request missing information could still be good; vagueness due to undefined pronouns/terms or references to unknown context does not make a User Request bad.

Reply:
IMO, none of these contribute meaningfully to further improvements in the benchmarks and some make the score a bit weaker. But the following:

```Determine whether the above User Request appears simple and straightforward.
A bad User Request will appear either: ambiguous in intent, complex, purposefully tricky, abnormal, or disgruntled.
A good User Request is phrased clearly and expresses an achievable intent. Basic conversational and non-propositional statements are also considered good.
Should an AI Assistant be able to properly answer the User Request, it is considered good. The AI Assistant handling this User Request has additional knowledge about: the user, domain-specific terms and abbreviations, and any necessary factual information. So a User Request missing information could still be good; vagueness due to undefined pronouns/terms or references to unknown context does not make a User Request bad.
```
seems to offer greater precision.


Benchmarks have been shared in slack, and I've set this prompt instead.

Reply:
Are there more changes we want to make to the wording?

Reply:
LGTM, thanks for re-benchmarking!